include/opt-sched/Scheduler/list_sched.h

```diff
@@ -52,6 +52,20 @@ class SequentialListScheduler : public ListScheduler {
   bool ChkInstLglty_(SchedInstruction *inst) const override;
 };
 
+// A list scheduler that schedules the instruction with the top heuristic value
+// Unalike ListScheduler this class considers instructions that are no ready
```

Comment:
I think there is a typo here: `ListScheduler this class considers instructions that are no ready`
Looks like the `no` needs to be removed.

Reply:
I fixed it